src/main/resources/schema.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE products
+(
+    id        BIGINT PRIMARY KEY AUTO_INCREMENT,
+    name      VARCHAR(255),
+    price DOUBLE,
+    image_url TEXT
```

Comment:
`TEXT` can be quite large. Check the actual length you need and consider changing it to a `VARCHAR` with an appropriate length.

Reply:
Most browsers like Chrome and Edge recommend a length of around 2000 characters 😊 So I think that should be fine!